grakn-test/src/test/java/ai/grakn/test/graql/reasoner/ReasoningTests.java

```diff
@@ -46,7 +48,7 @@ public void testSet10() {
         QueryBuilder iqb = testSet10.graph().graql().infer(true);
```

Comment:
It is not a test for testing sake. His tests test for different meanders of reasoning and I find them all very useful. He found some cases I didn't think of. But ok, I can come up with a more meaningful name.